src/app/shared/formly/types/checkbox-field/checkbox-field.component.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormGroup, ReactiveFormsModule } from '@angular/forms';
+import { FormlyModule } from '@ngx-formly/core';
+
+import { FormlyTestingComponentsModule } from 'ish-shared/formly/dev/testing/formly-testing-components.module';
+import { FormlyTestingContainerComponent } from 'ish-shared/formly/dev/testing/formly-testing-container/formly-testing-container.component';
+
+import { CheckboxFieldComponent } from './checkbox-field.component';
+
+describe('Checkbox Field Component', () => {
+  let component: FormlyTestingContainerComponent;
+  let fixture: ComponentFixture<FormlyTestingContainerComponent>;
+  let element: HTMLElement;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [CheckboxFieldComponent],
+      imports: [
+        FormlyModule.forRoot({
+          types: [{ name: 'checkbox', component: CheckboxFieldComponent }],
+        }),
+        FormlyTestingComponentsModule,
+        ReactiveFormsModule,
+      ],
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    const testComponentInputs = {
+      model: { active: true },
+      fields: [
+        {
+          key: 'active',
+          type: 'checkbox',
+        },
+      ],
+      form: new FormGroup({}),
+    };
+
+    fixture = TestBed.createComponent(FormlyTestingContainerComponent);
+    component = fixture.componentInstance;
+    element = fixture.nativeElement;
+
+    component.testComponentInputs = testComponentInputs;
+    fixture.detectChanges();
```

Comment:
I know I do this some places too, but the first fixture.detectChanges() should probably be in expect().not.toThrow()